app/v2/models/response/retrieveForeignPropertyAnnualSubmission/foreignProperty/ForeignPropertyAllowances.scala

```diff
@@ -21,10 +21,11 @@ import play.api.libs.json.{Json, OFormat}
 case class ForeignPropertyAllowances(annualInvestmentAllowance: Option[BigDecimal],
                                      costOfReplacingDomesticItems: Option[BigDecimal],
                                      zeroEmissionsGoodsVehicleAllowance: Option[BigDecimal],
-                                     propertyAllowance: Option[BigDecimal],
                                      otherCapitalAllowance: Option[BigDecimal],
-                                     structureAndBuildingAllowance: Option[BigDecimal],
-                                     electricChargePointAllowance:  Option[BigDecimal]
+                                     electricChargePointAllowance:  Option[BigDecimal],
+                                     zeroEmissionsCarAllowance: Option[BigDecimal],
+                                     propertyAllowance: Option[BigDecimal],
```

Comment:
`propertyIncomeAllowance` not `propertyAllowance` Maped from `propertyAllowance` downstream so will need custom reads